### PR TITLE
Fix monitoring via post requests

### PR DESCRIPTION
**Pull request by jjdlugosz**
Generally there are microservices which do support POST requests only. Selena didn't support such so far. There is an option in Selena to configure POST parameters, however to each POST request Selena sends one leading GET request. So every GET request finishes with an error. It causes monitoring not helpful. 

The change checks if administrator defines POST parameters. If so, Selena skips sending default GET request.

### CODE REVIEW

File: src/selena/services/monitoring.py [hunk 1]
```diff
@@ -88,43 +88,46 @@ def _get_test_cases(service, monitored_phrases=''):
...
-        'monitored_phrases': monitored_phrases,
-        'connection_timeout': service.connection_timeout,
-        'response_code': service.response_code,
-        'performance_issues_time': service.performance_issues_time,
-        'connection_timeout': service.connection_timeout,
-    }]
+    cases = []
     params = service.additionalrequestparam_set.all()
-    for param in params:
+    if len(params) > 0:
```
Comment (deejay1) [inline src/selena/services/monitoring.py]: just "if params:" should be enough, no need to check it's length

File: src/selena/services/monitoring.py [hunk 2]
```diff
@@ -136,42 +139,44 @@ def _get_test_cases(service, monitored_phrases=''):
...
-            'connection_timeout': service.connection_timeout,
-            'response_code': service.response_code,
-            'performance_issues_time': service.performance_issues_time,
-            'connection_timeout': service.connection_timeout,
-        })
-        for param in params:
+        if len(params) > 0:
+            for param in params:
+                cases.append({
+                    'url': '%s%s' % (
```
Comment (deejay1) [inline src/selena/services/monitoring.py]: maybe that dict can be extracted to an separate class/dict to avoid copy&paste?

**Comment by deejay1** [comment]
Sorry to comment again, but can you add a sphinx compatible doc string for this new function describing the function parameters? ;) Of course if you know what is going on there :P
Otherwise +1, it's much more readable now!

**Comment by deejay1** [comment]
+1

**Comment by h00dy** [comment]
+1

**Pull request merged**